.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -97,47 +134,60 @@ jobs:
     - name: Enable Node corepack
       uses: Lombiq/GitHub-Actions/.github/actions/enable-corepack@dev
 
-    - name: Restore NodeJS Packages (.NET)
+    # If runtime is defined, we need to add "--runtime=" to the string so it will be a valid build/publish option. The
+    # "build-dotnet" action requires the additional switches to be in separate lines (even the parameters), but we can
+    # take advantage of the dotnet CLI tolerating the usage of the equal sign.
+    - name: Set up runtime option
       run: |
-        if (Test-Path src/Utilities/Lombiq.Gulp.Extensions/Lombiq.Gulp.Extensions.csproj)
-        {
-          Write-Output "::group::Gulp Extensions found. It needs to be explicitly built before the solution."
-
-          $startTime = [DateTime]::Now
-          dotnet build src/Utilities/Lombiq.Gulp.Extensions/Lombiq.Gulp.Extensions.csproj --verbosity ${{ inputs.build-verbosity }}
-          $endTime = [DateTime]::Now
+        $runtime_option = "runtime-option="
 
-          Write-Output ("Gulp Extensions build took {0:0.###} seconds." -f ($endTime - $startTime).TotalSeconds)
-          Write-Output "::endgroup::"
+        if(![string]::IsNullOrEmpty("${{ inputs.runtime }}"))
+        {
+          $runtime_option = $runtime_option + "--runtime=" + "${{ inputs.runtime }}"
         }
 
-    - name: Build and Publish
-      if: ${{ inputs.runtime == '' }}
-      run: |
-        Invoke-Expression "${{ env.publish-command-base }}"
+        $runtime_option >> $env:GITHUB_ENV
 
-    - name: Build and Publish in Ready to Run mode
-      if: ${{ inputs.runtime != '' }}
-      run: |
-        Invoke-Expression "${{ env.publish-command-base }} --runtime ${{ inputs.runtime }} -p:PublishReadyToRun=true --self-contained true"
+    - name: Build and Static Code Analysis
+      uses: Lombiq/GitHub-Actions/.github/actions/build-dotnet@issue/NEST-316
+      with:
+        directory: ${{ inputs.build-directory }}
+        verbosity: ${{ inputs.build-verbosity }}
+        enable-code-analysis: ${{ inputs.build-enable-code-analysis }}
+        enable-nuget-caching: ${{ inputs.build-enable-nuget-caching }}
+        enable-npm-caching: ${{ inputs.build-enable-npm-caching }}
+        cache-version: ${{ inputs.build-cache-version }}
+        solution-or-project-path: ${{ inputs.web-project-path }}
+        dotnet-build-switches: |
+          --self-contained
+          ${{ inputs.self-contained }}
+          -p:PublishReadyToRun=${{ inputs.ready-to-run }}
+          ${{ env.runtime-option }}
 
+    - name: Publish
+      run: >
+        dotnet publish (Get-ChildItem ${{ inputs.web-project-path }}).FullName --configuration Release --no-build
+        --output '${{ inputs.build-directory }}/Published' --verbosity ${{ inputs.build-verbosity }}
+        --self-contained ${{ inputs.self-contained }} -p:PublishReadyToRun=${{ inputs.ready-to-run }}
+        ${{ env.runtime-option }}
```

Comment:
Indent these and have one line per option for clarity.

Reply:
I can't indent them, since it will not recognize the switches: 
![image](https://user-images.githubusercontent.com/80963259/191996387-790e2bcf-d114-417c-a6a7-75880145c733.png)

Reply:
You need to write PS here, so use backticks to divide single-line commands into multiple lines.